### DISCUSSION TITLE
Is there a good method of extracting tags per component?

### DISCUSSION THREAD

**Discussion by finleywilliamsonV** [marcioggs/ng-i18n-merge-files#11] [Q&A]
With Angular's tooling, you use `ng extract-i18n` to extract all the translations into a messages file, which then is used to create the message files for various languages.

If this package compiles disperse translation files into one place, does one have to extract all tags at the component level manually?

Thanks for the help!

**Comment by marcioggs** [comment]
Hi,
Thanks for using the package.
Yes, one has to extract the tags as you've described.

**Reply by finleywilliamsonV** [reply]
Sure was just wondering, thanks for the response :)
